docs/tutorials/03-publishing-pilets.md

```diff
@@ -19,52 +19,121 @@ We designed Piral to be as simple as possible, while remaining as flexible and p
 
-On the feed details page we get all sorts of information. Most notably, we see the currently available pilets. However, before we can publish a pilet, we need to obtain an API key. For this we can go to the API keys of the current feed (either from the homepage listing all feeds, or on the details page by clicking the keys icon on the feed bar).
+## Creating a Feed in the Piral Feed Service
+For publishing (or retrieving) pilets you will need a feed. A feed is like a container or folder, which contains a set of pilets. This section describes how to setup a feed for serving Pilets.
 
-The API key management page lists all currently active API keys. It does not list the actual API keys. These are only visible when they are created.
+### Access to the Feed Service
+To use the community edition of the **feed service** as part of the Piral Cloud Services, you don't need to create or register an account. Just navigate to the community feed service <https://feed.piral.io> and sign in with a Microsoft Account by pressing the sign-in button on the home page of the service.
 
-![Available API Keys](../diagrams/api-keys.png)
+![Microsoft Login](../diagrams/ms-login.svg)
 
-By clicking the "generate API key" button we can provide all necessary information for generating a new API key.
+Upon successful login the landing page of the feed service will be displayed.
 
-Once a new key has been generated it will be shown for the first - and last - time. Make sure to copy it to a secure location. If you lose the key you should revoke it such that also no one else will be able to use it.
+### Create a feed
+
+For creating a new feed, follow the link `Cerate Feed` on the right-hand side of the top menu.
+
+![Creating a new feed](../diagrams/creating-feed.png)
+
+To create the feed, we need to give it a proper (unique) name. The system will check if the provided name is still available. The description can only be seen by you - it is optional. The allowed hosts you can leave blank for now (resolving to `*`).
+
+When you are done, just press `CREATE` and wait for the operation to complete. You will now see the feed details page, which shows you the available pilets.
+
+![Feed Overview](../diagrams/feed-overview.png)
+
+In our case we will see at least the feed, which we just have created. 
+
+## Obtain an API Key
+
+Before we can publish our first pilet to our newly created feed, we need to obtain an API key.
+
+### Create an API Key
+To see al list of all available API keys, invoke the link `Manage API Keys` or press the "key" icon on the right-hand side of the the feed bar of the current feed. As we did not create an API key, the list will be empty.
+
+By clicking the `Generate API Key` button we can provide all necessary information for generating our first API key.
+
+![Generate API Keys](../diagrams/new-api-key.png)
+
+Upon pressing the `GENERATE` button, a new API key will be created.
```

Comment:
I would just write "Generate" (like above). The full upper case is a notion of styling and not content (otherwise, we need to change the `Generate API Key` to be uppercase, too).

Reply:
Updated